--- gqcam-0.9.1/timer.c~	2011-12-29 23:32:33.000000000 +0200
+++ gqcam-0.9.1/timer.c	2011-12-29 23:33:14.636960876 +0200
@@ -81,7 +81,7 @@
 
   camera->on_timer = 1;
   
-  if(strnlen(gtk_entry_get_text(GTK_ENTRY(camera->timer_struct.commandentry))) != 0){
+  if(strlen(gtk_entry_get_text(GTK_ENTRY(camera->timer_struct.commandentry))) != 0){
     strcpy(camera->timer_struct.command, gtk_entry_get_text(GTK_ENTRY(camera->timer_struct.commandentry)));
     camera->timer_struct.iscommand = 1;
   }
--- gqcam-0.9.1/jpeg.c~	2005-04-04 04:25:09.000000000 +0300
+++ gqcam-0.9.1/jpeg.c	2011-12-29 23:43:48.739527083 +0200
@@ -1,6 +1,7 @@
 /* jpeg decompression code added from libjpeg with original comments */
 
 #include <stdio.h>
+#include <string.h>
 
 #include "jpeg.h"
 #define SIZEOF(object)	((size_t) sizeof(object))
